recipes-kernel/linux/linux-mainline/openvario.cfg

```diff
@@ -105,19 +99,13 @@ CONFIG_SND_AUDIO_GRAPH_CARD=m
 # CONFIG_HID_REDRAGON is not set
 # CONFIG_HID_MICROSOFT is not set
 # CONFIG_HID_MONTEREY is not set
-CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_XHCI_HCD=m
 CONFIG_USB_XHCI_PLATFORM=m
 CONFIG_USB_MUSB_SUNXI=y
-CONFIG_USB_ACM=m
```

Comment:
Are you sure this will not break USB-serial devices that register as `/dev/ttyACM0`?

Reply:
Which one does? Does it pretend to be a USB modem, instead of a serial port? (I asked about USB modem support on https://github.com/Openvario/meta-openvario/discussions/185)

Reply:
At least one device I can recall that does that are Arduino boards. Not sure why they do that - they are not modems, but appear as `ttyACM0` when you connect them. I think I saw more of them but can't recall what it was. To me it's a mystery why some USB-serial devices are `ttyUSB0` and some `ttyACM0`, but I think both are pretty useful for OV.

Reply:
OK, dropping this part in the next push.